src/Consoles/Messages.cs

```diff
@@ -179,18 +170,118 @@ public override void ProcessKeyboard(Keyboard info)
                 PointerIndex += 1;
             else if (info.IsKeyPressed(Keys.Up))
                 PointerIndex -= 1;
+        }
+    }
+
+    [AttributeUsage(AttributeTargets.All)]
+    public class DisplayAttribute : Attribute
+    {
+        public string Title { get; }
+
+        public DisplayAttribute(string title)
+        {
+            Title = title;
+        }
+    }
+
+    public class QuestionMessage : ChoiceMessage<Enum>
+    {
+        public QuestionMessage(Type answersType, string question = null)
+            :base(answersType.GetEnumValues()
+                .Cast<Enum>()
+                .Select(@enum => new Tuple<Enum, string>(@enum, @enum.GetAttributeOfType<DisplayAttribute>()?.Title))
+                .Where(tp => tp.Item2 != null)
+                , question)
+        {
+            if (question != null)
+                Rows = 2;
+            else
+                Rows = 3;
```

Comment:
W sumie mogę to zmienić tak, żeby liczba wierszy były automatycznie liczona z wysokości konsoli